testbed/testbed/load_generator_test.go

```diff
@@ -0,0 +1,58 @@
+package testbed
+
+import (
+	"testing"
+	"time"
+)
+
+func Test_perWorkerTickDuration(t *testing.T) {
+	for _, test := range []struct {
+		name                                          string
+		expectedTickDuration                          time.Duration
+		dataItemsPerSecond, itemsPerBatch, numWorkers int
+	}{
+		// Because of the way perWorkerTickDuration calculates the tick interval using dataItemsPerSecond,
+		// it is important to test its behavior with respect to a one-second Duration in particular.
+		{
+			name:                 "less than one second",
```

Comment:
Since this change is to resolve the divide by zero risk, we should have a test case that would have previously resulted in the divide by zero panic.

Reply:
Thanks for the review @dehaansa! One of the test cases was as you describe (indeed it was the set of configuration that caused me to find this bug in the first place). I've made the comment and test name more explicit on this point.